File: pinyougou-web-shop/src/main/webapp/js/service/seckillGoodsService.js
//服务层
app.service('seckillGoodsService',function($http){

    //搜索
    this.search=function(page,rows,searchEntity){
        return $http.post('../seckillGoods/search.do?page='+page+"&rows="+rows, searchEntity);
    };

    //增加
    this.add=function(entity){
        return  $http.post('../seckillGoods/add.do',entity );
    }
});

File: pinyougou-interface/src/main/java/cn/itcast/core/service/SeckillGoodsService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.seckill.SeckillGoods;
import entity.PageResult;

public interface SeckillGoodsService {
    PageResult search(Integer page, Integer rows, SeckillGoods goods);

    SeckillGoods findOne(Long id);

    void updateStatus(Long id, String status);

    void delete(Long id);
}
